src/include/migraphx/operation.hpp

```diff
@@ -56,6 +56,8 @@ struct operation
     std::string name() const;
     /// An optional method that can be used to finalize the operator before running
     void finalize(context& ctx);
+    /// An optional method that can be used to set offload copy for a target
+    bool runs_on_offload_target();
```

Comment:
Make this either take bool arg to set the state of the offload target run, and rename this. The current function name reads like a getter (like "does this run on offload target") rather than setting the run state on the offloaded target.

Reply:
It is supposed to be a getter. and not a setter.

Reply:
remove the comment above then, it doesnt make sense to what that functions intent is